public/index.html

```diff
@@ -9,6 +9,13 @@
     <meta name="theme-color" content="#000000" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    {% if should_add_telemetry %}
+    <script type="text/javascript">   
+      window.heap=window.heap||[],heap.load=function(e,t){window.heap.appid=e,window.heap.config=t=t||{};var r=document.createElement("script");r.type="text/javascript",r.async=!0,r.src="https://cdn.heapanalytics.com/js/heap-"+e+".js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(r,a);for(var n=function(e){return function(){heap.push([e].concat(Array.prototype.slice.call(arguments,0)))}},p=["addEventProperties","addUserProperties","clearEventProperties","identify","resetIdentity","removeEventProperty","setEventProperties","track","unsetEventProperty"],o=0;o<p.length;o++)heap[p[o]]=n(p[o])};
+      heap.load("{{ heap_app_id }}"); 
+      heap.identify("{{ heap_user_identity }}");
```

Comment:
Hmm we don't have `heap.identify` on kedro docs for example - why do you need this?

Reply:
In viz, we need to match the user on the web with the user on the command line. The CLI user is identified by the host machine so I pass the same hash to the client. In the doc, we don't have the command line so we just let Heap work out who it is based on the session.

Reply:
Makes sense, thank you!